bin/okc-psql

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-docker run -it --rm --net host postgres psql -h localhost -U postgres
+docker run -it --rm --net host postgres psql --command "CREATE DATABASE okcollege_dev;" -h localhost -U postgres
```

Comment:
this file is for debugging the database, not creating a table in the database.

better with a separate bin file for the database creation or embed it directly in the docker-compose